Elevel/Elevel.Api/Controllers/FileController.cs

```diff
@@ -40,10 +41,13 @@ public IActionResult Upload([Required] List<IFormFile> formFiles)
         /// <param name="filePath">The path of file</param>
         /// <returns></returns>
         [HttpGet(nameof(Download))]
-        public IActionResult Download([Required] string filePath)
+        public FileResult Download([Required] string filePath)
         {
-            var file =  _fileService.DownloadFile(filePath);
-            return File(file.Content, file.ContentType, file.FileName);
+            var file = _fileService.DownloadFile(filePath);
+            return new FileStreamResult(file.Stream, file.ContentType)
+            {
+                FileDownloadName = file.FileName,
+            };
```

Comment:
FileDownloadName = _fileService.DownloadFile(filePath).FileName,

Reply:
If that, we need to request DownloadFile at least one more time (for file.Stream and/or file.ContentType), and that would be not good for memory/time/unexpected errors, imho